cadquery/cq.py

```diff
@@ -61,6 +62,7 @@
 
 CQObject = Union[Vector, Location, Shape, Sketch]
 VectorLike = Union[Tuple[float, float], Tuple[float, float, float], Vector]
+CombineMode = Union[bool, Literal["cut", "a", "s"]]  # a : additive, s: subtractive
```

Comment:
I added also modes used in Sketch. Note that it is better to use Literal for typing.